src/main/java/com/kodilla/ecommercee/service/CartService.java

```diff
@@ -36,4 +51,28 @@ public void deleteCart(Long cartId) {
         cartRepository.deleteById(cartId);
     }
 
+    public Item getItem(Long itemId) {
+        return itemRepository.getOne(itemId);
+    }
+
+    public void addItem(Item item) {
+        if(item.getCart().getId().equals(null) || item.getProduct().getId().equals(null)){
```

Comment:
Jak uda Ci się wybrnąć z zapętlania, to przy dodawaniu nowego Item musisz uwzględnić:
- cena item = ilość produktu * cena produktu
Dopiero potem robisz save